tests/fixtures/Notification/notification-receive-test.yml

```diff
@@ -0,0 +1,5 @@
+-
+  notification: notification-status-change.xml
+  expectations:
+    assertions:
+      - notification.getNotificationType() === 'RequestStatusChange'
```

Comment:
May be it would be nice to add little bit more cases.

Reply:
Ok I've added 2 others. Here it's more just a proof that they're in place and can be developed.